src/Listener/ApiQueryLogListener.php

```diff
@@ -106,6 +106,10 @@ protected function _getSources()
      */
     protected function _getSource($source)
     {
-        return ConnectionManager::get($source);
+        try {
+            return ConnectionManager::get($source);
+        } catch (\Cake\Datasource\Exception\MissingDatasourceConfigException $e) {
```

Comment:
rather want these to lines in try block? https://github.com/Apacta/crud/blob/edf3a9e1a2d3e1523429e72f75d7fb7b75aff259/src/Listener/ApiQueryLogListener.php#L51-L52